src/algorithm/frames.hxx

```diff
@@ -126,6 +126,7 @@ namespace pinocchio
     if(rf == WORLD)
     {
       getJointJacobian(model,data,joint_id,WORLD,PINOCCHIO_EIGEN_CONST_CAST(Matrix6xLike,J));
+      return;
```

Comment:
Is there a reason to have the return here? If that's part of the pinocchio coding style convention to have `returns` early in a function, then there should also be a `return` for the `else if(rf == LOCAL || rf == LOCAL_WORLD_ALIGNED) ` branch.

Reply:
You removed it in one of your commits. I've just restored it.
Concerning the convention, it is just helpful for the compiler, and to avoid checking other flags. Similar to the break instruction in a switch.